### ISSUE TITLE
c0d3 doesn't work without javascript

### ISSUE THREAD

**Issue by Ulisseus** [garageScript/c0d3-app#508]
Eternal loading screen.
It would be nice if at least landing page and lessons would be usable without js since they're almost fully static.
